.angular-cli.json

```diff
@@ -21,7 +21,8 @@
       "prefix": "app",
       "serviceWorker": true,
       "styles": [
-        "styles.scss"
+        "styles.scss",
+        "../node_modules/font-awesome/scss/font-awesome.scss"
```

Comment:
Please use import in `styles.scss` like https://github.com/ReactiveX/rxjs-docs/blob/master/src/styles.scss#L1

Reply:
Hmm, I'm getting build errors when I try to import it like this:

`@import '~font-awesome/scss/font-awesome';`

Any tips on how I should import it using this technique?

Reply:
I will check

Reply:
Got it, using this: https://stackoverflow.com/a/34366930/2521893

Reply:
Both of these work it's not able to resolve the fonts which is why it's failing.
`@import '~font-awesome/css/font-awesome.min.css';`
`@import './../node_modules/font-awesome/css/font-awesome.min.css';`

Reply:
It seems to work after these changes, doesn't it? fc9ca536966cc18f7283eb833232eabf3db3db35

```
$fa-font-path: "~font-awesome/fonts";
@import "~font-awesome/scss/font-awesome";
```